src/main/java/dk/kb/util/yaml/YAML.java

```diff
@@ -740,16 +744,32 @@ private static List<String> removeTraversedEntry(List<String> yPath) {
         }
     }
 
+    /**
+     * Create a new instance of the input list of yPath elements, where the first element has been substituted.
+     * @param yPath list of elements from a YAML path.
+     * @param newEntry a new entry, which is to replace the first entry in {@code yPath}.
+     * @return a new List containing all values from yPath, except the first, which has been substituted with {@code newEntry}.
+     */
+    private static List<String> replaceFirstEntryInYPath(List<String> yPath, String newEntry){
```

Comment:
Not used anymore. Please remove.

Reply:
Addendum: There are multiple other unused methods in the file. Please remove those too.

Reply:
Only if they have been added in this pull request (to maintain backwards compatibility) and if they are not generally usable. A quick search says that `getMultipleFromSubYaml`, which I would suggest removed as it is far too specialized for general use.

Reply:
Also: `getMultipleFromSubYamlList` (note the `List` suffix) is not used any more and is `private`.

Reply:
> Only if they have been added in this pull request (to maintain backwards compatibility) and if they are not generally usable. A quick search says that `getMultipleFromSubYaml`, which I would suggest removed as it is far too specialized for general use.

This was added earlier and is in use in the maven-plugin I've written earlier. My guess is that this can and should be removed when the maven plugin has been updated to make use of the new implementation